app/dashboard/templates/profiles/profile.html

```diff
@@ -262,17 +174,79 @@
 
     <script type="text/javascript">
       $(document).ready(function() {
-        if ($('.profile-bounties--tips .bounty_row--hidden').length > 0) {
-          var $button = $('.profile-bounties--tips .profile-bounties__btn-show-all');
-          $button.removeClass('hidden');
-          $button.on('click', function(event) {
-            $(this).remove();
-            $('.profile-bounties--tips .bounty_row').removeClass('bounty_row--hidden');
-          });
-        }
         setupTabs('#activity-tabs'); 
 
+        const tabSection = document.querySelector('#activity-tabs-sections');
+        const updateViewBtn = document.querySelector('#update-view-btn');
+        let fetchInProgress = false;
+
+        function updateView(ignoreScrollOffset) {
+          window.setTimeout(updateView, 300);
+
+          if (fetchInProgress) {
```

Comment:
this seems like a lot of state management // pretty large function..

Reply:
Well, I actually want to re-use this code because I have a bounty row results refactor on the back of my mind. I'm just on hold because of all the stuff you guys are doing right now 😄 .

Basically I want to unify the rendering of the bounty query results (across dashboard and retail) with django templates and pagination.
Right now we have different 'out of sync' templates and the dashboard bounty rendering is done via some js rendering template - I wan't to get rid of this 😄 Thoughts?

Reply:
@mbeacom what do u think